sailthru_content/services/tests/test_sailthru_translation_service.py

```diff
@@ -1,12 +1,15 @@
+from __future__ import unicode_literals
 from faker import Factory
 
```

Comment:
Remove the newline. All three of these are third-party libraries. There _should_ be a new line after the `__future__` import.